src/java.base/share/man/java.1

```diff
@@ -5533,6 +5533,37 @@ application start\-up script with the same technique:
 .PP
 \f[CB]env\ JAVA_TOOL_OPTIONS=\-XX:+RecordDynamicDumpInfo\ bash\ app_start.sh\f[R]
 .RE
+.SS Using Dynamic CDS Archive File with \-XX:+AutoCreateSharedArchive
```

Comment:
Should the above be "Creating/Using Dynamic CDS Archive..."?

Reply:
Actually my first version was "Creating/Using Dynamic CDS Archive...", but this line (section title) is too long and gets badly wrapped when the man page is viewed in a terminal using `nroff`.  So I dropped the word "Creating" as the meaning should still be clear enough.

Reply:
Ok. Approving as is.

Reply:
Based on the previous two subsections "Creating" would be the consistent way to express this.